packages/sanes-chrome-extension/src/extension/messages.ts

```diff
@@ -23,12 +24,15 @@ export interface MessageToBackground {
   readonly data?: any;
 }
 
-export interface CreatePersonaResponse {
+export interface PersonaData {
   readonly accounts: ReadonlyArray<AccountInfo>;
   readonly mnemonic: string;
   readonly txs: ReadonlyArray<ProcessedTx>;
 }
 
+/* eslint-disable-next-line @typescript-eslint/no-empty-interface */
+export interface CreatePersonaResponse extends PersonaData {}
```

Comment:
Why this?

Reply:
To be able to reuse `PersonaData` but have an explicit return type for `sendCreatePersonaMessage` analogue to `sendGetPersonaMessage`, which can change independently of `PersonaData`.